src/docs/02_architecture_constraints.adoc

```diff
@@ -17,3 +17,36 @@ If needed you can subdivide them into
 technical constraints, organizational and political constraints and
 conventions (e.g. programming or versioning guidelines, documentation or naming conventions)
 ****
+
+=== Technical constraints
+
+[options="header",cols="1,2"]
+|===
+|Constraint|Explanation
+| Solid | The application must follow the link:https://github.com/solid/solid-spec[Solid specifications] and make use of Solid pods, as well as being decentralized and follow linked data principles.
+| React framework | React is the framework that will be used to implement the application. The architecture of the application has to be adapted to the inner workings of the Javascript library (components, routing, rendering...).
+| Interoperability between similar webapps | The design for the data saving and retrieving has to be thoughtfully taken care of such that a user can switch between similar apps without having to modify their data (using of Solid, W3C standards, etc.).
+| Test-Driven Development | Any functionality added must be constantly and thoroughly tested by the development team, making use of the testing tools available (Cucumber, Travis, Codecov).
+|===
+
+=== Organizational and political constraints
+
+[options="header",cols="1,2"]
+|===
+|Constraint|Explanation
+| Short development time | Due to the nature of the course, the development time for the "finished" product is extremely small, consisting only of seven weeks.
+| Almost inexistent experience | The team has hardly any experience with the technologies planned for use. This severely limits the productiveness of each member, and the team.
```

Comment:
I would not consider this a constraint. Although this limits the capability of the development for the app, it is not a limitation in the sense of [designing the architecture or taking development decisions](https://docs.arc42.org/section-2/).

Reply:
I feel it does affect design and development decisions: given the lack of experience and short time frame, it may be wise to incur in technical debt to kickstart the work.

Reply:
Fair enough then, we will take this into account as a constraint.